src/device-registry/utils/create-device.js

```diff
@@ -153,9 +153,22 @@ const createDevice = {
         return {
           success: false,
           message: "creation is not yet possible for this organisation",
-          status: HTTPStatus.NOT_IMPLEMENTED,
+          status: httpStatus.NOT_IMPLEMENTED,
         };
       }
+
+      if (process.env.NODE_ENV !== "production") {
+        return {
```

Comment:
in case it is not production ENV, advise the user to use SOFT creation endpoint. @FaithDaka @BenjaminSsempala